msibi/workers.py

```diff
@@ -48,13 +48,17 @@ def _hoomd_worker(args):
     state, idx, gpus = args
     log_file = os.path.join(state.state_dir, 'log.txt')
     err_file = os.path.join(state.state_dir, 'err.txt')
+    if state.HOOMD_VERSION == 1:
+        executable = 'hoomd'
+    elif state.HOOMD_VERSION == 2:
+        executable = 'python'
     with open(log_file, 'w') as log, open(err_file, 'w') as err:
         if gpus:
             card = gpus[idx % len(gpus)]
-            cmds = ['mpiexec', '-n', '1', 'hoomd', 'run.py', '--gpu={card}'.format(**locals())]
+            cmds = ['mpiexec', '-n', '1', executable, 'run.py', '--gpu={card}'.format(**locals())]
         else:
             logging.info('    Running state {state.name} on CPU'.format(**locals()))
-            cmds = ['hoomd', 'run.py']
+            cmds = [executable, 'run.py']
```

Comment:
That seems like an easy way to start since it appears that you were only launching one rank per sim anyways, right? This shouldn't have been a major limitation for the system sizes you've been working on.